### PR TITLE
[14.0][FIX] account fiscal year fix

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
based on : https://github.com/OCA/account-financial-tools/pull/1264 <br/>
less right needed

### CODE REVIEW

**Review by MiquelRForgeFlow — APPROVED** [review]

File: account_fiscal_year/security/ir.model.access.csv [hunk 1]
```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_account_fiscal_year_user,account.fiscal.year.user,model_account_fiscal_year,account.group_account_user,1,0,0,0
-access_account_fiscal_year_manager,account.fiscal.year.manager,model_account_fiscal_year,account.group_account_manager,1,1,1,1
+access_account_fiscal_year_manager,account.fiscal.year.manager,model_account_fiscal_year,account.group_account_invoice,1,1,1,1
```
Comment (AaronHForgeFlow) [inline account_fiscal_year/security/ir.model.access.csv:3]: Is that actually needed?

Comment (Cedric-Pigeon) [reply account_fiscal_year/security/ir.model.access.csv:3]: yes otherwize you can't create an invoice

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LGTM

**Review by HaraldPanten — APPROVED** [review]

**Review by kittiu — APPROVED** [review]
LGTM

**Comment by kittiu** [comment]
Sorry, don't we need to depends on server-ux's `date_range` ?
I don't see it on dependencies.

**Comment by AaronHForgeFlow** [comment]
> Sorry, don't we need to depends on server-ux's `date_range` ? I don't see it on dependencies.

I think so. It should be included.

**Comment by rafaelbn** [comment]
Hello @xavier-bouquiaux , please could you check this comment https://github.com/OCA/account-financial-tools/pull/1265#issuecomment-966809093?

Thank you!

**Comment by xavier-bouquiaux** [comment]
> Hello @xavier-bouquiaux , please could you check this comment [#1265 (comment)](https://github.com/OCA/account-financial-tools/pull/1265#issuecomment-966809093)?
> 
> Thank you!

Done

**Comment by AaronHForgeFlow** [comment]
Sorry, this should have been a major merge, as a new dependency is added.

**Pull request merged**